profile-service/src/main/resources/db/migration/common/V9.28__add_post_delete_permission.sql

```diff
@@ -0,0 +1,19 @@
+INSERT INTO `Role` (`name`)
+VALUES ('PostDelete')
+ON DUPLICATE KEY UPDATE `name` = `name`;
+
+INSERT INTO `Permission` (`name`, `type`, `description`, `principal`, `resource`, `actions`, `effect`)
+VALUES
+    ('post:delete',
+     'POST',
+     'Can delete posts',
+     'tis:profile::user:',
+     'tis:post::delete:',
```

Comment:
I'm not sure if it matters, but "tis:post::delete:" doesn't look like a resource to me.

Reply:
Yeah agree. It seems like an action rather than a resource.

Reply:
@hihilary What is your suggestion? Can I use 'tis:post::*' or 'tis:post::posts'? Please let me know.

Reply:
Maybe "tis:posts::post:" ? I've seen a resource "tis:posts:-:post:" is used in Profile Permission table, but not sure which one is correct TBH.

Reply:
Changed it to 'tis:posts::post:'